src/lib/json-big.js

```diff
@@ -0,0 +1,26 @@
+import JsonBig from '@aeternity/json-bigint';
+import BigNumber from 'bignumber.js';
+
+const jsonBig = JsonBig({ storeAsString: true });
+
+const convertValuesToBigNumbers = (value) => {
+  if (typeof value === 'object' && value !== null && value.constructor === Object) {
+    return Object.entries(value)
+      .map(([key, _value]) => [key, convertValuesToBigNumbers(_value)])
+      .reduce((p, [k, v]) => ({ ...p, [k]: v }), {});
+  }
```

Comment:
Could we remove the usage of map here?
```suggestion
    return Object.entries(value)
      .reduce((p, [k, v]) => ({ ...p, [k]: convertValuesToBigNumbers(v) }), {});
  }
```